content/200-orm/200-prisma-client/100-queries/037-relation-queries.mdx

```diff
@@ -1286,7 +1286,7 @@ const postsByUser = await prisma.post.findMany({
 
 The main difference between the queries is that the fluent API call is translated into two separate database queries while the other one only generates a single query (see this [GitHub issue](https://github.com/prisma/prisma/issues/1984))
 
-> **Note**: You can use the fact that `.findUnique({ where: { email: 'alice@prisma.io' } }).posts()` queries are automatically batched by the Prisma dataloader to [avoid the n+1 problem in GraphQL resolvers](/orm/prisma-client/queries/query-optimization-performance#solving-n1-in-graphql-with-findunique-and-prismas-dataloader).
+> **Note**: You can use the fact that `.findUnique({ where: { email: 'alice@prisma.io' } }).posts()` queries are automatically batched by the Prisma ORM dataloader to [avoid the n+1 problem in GraphQL resolvers](/orm/prisma-client/queries/query-optimization-performance#solving-n1-in-graphql-with-findunique-and-prisma-orms-dataloader).
```

Comment:
data loader is part of Prisma Client

Reply:
Let's say "Prisma dataloader in Prisma Client"

Reply:
Ok I also changed the link so there may be a broken link on another file, I'll keep an eye on the build output